explorer_node:
  ros__parameters:
    robot_base_frame: base_link
    map_topic: map
    costmap_topic: global_costmap/costmap
    visualize: true
    planner_frequency: 1
    progress_timeout: 15.0
    potential_scale: 1.0
    gain_scale: 0.0
    transform_tolerance: 1.0
    min_frontier_size: 0.3
    max_planning_size: 5
    how_many_corners: 8
    w_euc_cost: 1.0
    w_traversal_cost: 2.0
